pgc_ortho.py

```diff
@@ -192,8 +192,33 @@ def main():
             ortho_functions.formats[args.format]
         ))
 
-        done = os.path.isfile(dstfp)
-        if done is False:
+        # Must be a better way to check this
```

Comment:
Check out the PR I made into your branch Brian, showing how we can refactor this to simplify the code: https://github.com/PolarGeospatialCenter/imagery_utils/pull/32